src/components/Dashboard.vue

```diff
@@ -71,9 +79,14 @@ export default {
 
 		getItemTargetUrl() {
 			return (note) => {
-				return generateUrl('/apps/notes/note/' + note.id)
+				return generateUrl(`/apps/notes/note/${note.id}`)
 			}
 		},
+
+		buttonsFooterStyle() {
+			const marginTop = this.items.length > 0 ? 20 + (this.displayedItemsCount - this.items.length) * 60 : 10
+			return { marginTop: `${marginTop}px` }
+		}
```

Comment:
This looks quite complicated. You are comparing to the tasks app, but I don't see such a complicated calculation, there. Do we really need this or can we copy the styling from the tasks app?

Reply:
The main motive behind this implementation was to have the button stay at the bottom of the widget, rather than below the content. In Tasks, the button stays below the content :

| | |
|--------------|-----------|
| <img alt="Tasks Widget" src="https://user-images.githubusercontent.com/12085737/178208674-f4eda455-160d-42a6-864b-cbc74bcc4805.png" width="500"> | <img alt="Notes Widget" src="https://user-images.githubusercontent.com/12085737/178122918-ac7cf860-1cb2-4a5d-8474-d98dec4a93c0.png" width ="500"> |

I'd argue having the button at a fixed height is better, but we could as well simplify the logic and have the button stay below the content.